content/blog/botan-first-milestone.md

```diff
@@ -0,0 +1,248 @@
++++
+title = "Botan: The first milestone"
+description = "A step towards improving the state of Haskell cryptography"
+date = 2024-06-03T12:15:00
+updated = 2024-06-03T12:15:00
+draft = false # Leave this as true so that it is not published
+template = "blog/page.html"
+
+[extra]
+lead = "After 8 months of work, we've reached an important milestone. Now we're here to tell you about what is coming next."
++++
+
+# Botan: The First Milestone
+
+What are you using cryptography for? Would you be interested in trying something new? Perhaps an alternate backend for your cryptography needs?
+
+We're working on `botan`, a cryptographic kitchen sink, and after 8 months of work, we've reached an important milestone: we've successfully published the first released version `0.0.1` of  [botan-bindings][botan-bindings] and [botan-low][botan-low], and `botan` has made it to [package candidate][botan] status!
+
+Now we are hard at work on getting the first release version of `botan` ready too, and could use your feedback!
+
+## What is Botan?
```

Comment:
I would suggest that this sub-header be removed and that its content appears right next to `# Botan: the first milestone".

Reply:
I've done my best to juggle things around again, but it conflicts with your suggestion of placing the "What are you using cryptography for?" hook at the beginning. I've followed the following logic for the revised first section:

- Hook has to come first, its the definition of a hook
- Author intro  has to happen immediately after the hook, it would be first otherwise
- Summary ("What is this blog about? It is about botan's milestone) introduces topic (milestone) and subject (botan)
- Explanation ("What is botan") has to come after summary and topic / subject introductions
- "Summarize, then Explain" is an important rule for me because otherwise I easily become impossible to follow

Effectively it embodies the following interaction:

"Are you looking for ____? I am ____ and I would like to talk to you about how ____ can help. What is _____? _____ is ...."